### PR TITLE
Prep javascript arrays - James Aguirre

### PR DESCRIPTION
**Pull request by james-aguirre**
Screenshot 2023-03-02 at 11 01 10 PM

### CODE REVIEW

File: prep-javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgress'];
+console.log(languages);
+const firstElement = languages[0];
+console.log(firstElement);
+const thirdElement = languages[2];
+console.log(thirdElement);
+const length = languages.length;
+console.log(length);
+const lastElement = languages[5];
```
Comment (BrettAlbright) [inline prep-javascript-arrays/main.js]: make sure you make use of the `length` variable when accessing the last index of the array

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
great work! just make sure to make use of the `length` variable when trying to access the last index of the array

File: prep-javascript-arrays/main.js (hunk 1 above)
Comment (james-aguirre) [reply prep-javascript-arrays/main.js]: @Zhiwutian Is this correct?
Screenshot 2023-03-03 at 10 10 44 PM

File: prep-javascript-arrays/main.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgress'];
+console.log(languages);
+const firstElement = languages[0];
+console.log(firstElement);
+const thirdElement = languages[2];
+console.log(thirdElement);
+const length = languages.length;
+console.log(length);
+const lastElement = length - 1;
```
Comment (BrettAlbright) [inline prep-javascript-arrays/main.js]: here you want to get the last element in the array by targeting the array at the index which is the length of the array minus one. `postgres` is the last element value here.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small update needed!

File: prep-javascript-arrays/main.js (hunk 2 above)
Comment (james-aguirre) [reply prep-javascript-arrays/main.js]: @Zhiwutian Is this correct?
Screenshot 2023-03-04 at 10 11 50 AM

Screenshot 2023-03-04 at 10 11 58 AM

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**